openid-sharedsignals-framework-1_0.md

```diff
@@ -564,11 +535,44 @@ verification_endpoint
 
 critical_subject_members
 
-> OPTIONAL. List of member names in a Complex Subject which, if present in
+> OPTIONAL. An array of member names in a Complex Subject which, if present in
   a Subject Member in an event, MUST be interpreted by a Receiver.
+  
+supported_scopes
+
+> OPTIONAL. A list of OAuth {{RFC6749}} scope names that the Transmitter supports for specific endpoints. The value of this field is a JSON object that has the endpoint names as keys, and arrays of scope name strings they support as their values. OAuth tokens obtained using any of the scopes defined here MUST be accepted by the specified endpoint. Any key that is not defined as an endpoint in the Transmitter Configuration Metadata MUST be ignored
+
+authorization_servers
+
+> OPTIONAL. An array supported authorization servers and the scopes they support. Each element of the array is a Authorization Server Descriptor JSON object defined in the section {{authz-server-descriptor}} below.
 
 TODO: consider adding a IANA Registry for metadata, similar to Section 7.1.1 of
-{{OAUTH-DISCOVERY}}. This would allow other specs to add to the metadata.
+{{RFC8414}}. This would allow other specs to add to the metadata.
+
+### Authorization Server Descriptor {#authz-server-descriptor}
+An Authorization Server Descriptor is a JSON object that has two keys:
+
+scopes
+
+> REQUIRED. An array of scope names supported by the authorization server
+
+servers
+
+> REQUIRED. An array of authorization server URLs. This is the URL from which the Authorization Server Metadata MAY be obtained by following the process described in Section 3 of RFC8414 {{RFC8414}}
```

Comment:
I fear the section numbers in RFC8414 may change in potentially subsequent drafts. Not sure what's the ideal way to reference "Obtaining Authorization Server Metadata"

Reply:
I don't think RFCs can change. I think they create new RFC numbers if a change is required.